src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsNuGetFramework.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>A type that represents the components of a .NET Target Framework Moniker.</summary>
+    /// <remarks><c>System.Runtime.Versioning.FrameworkName</c> does not support .NET 5 Target Framework Monikers with a platform, but this type does.</remarks>
+    public interface IVsNuGetFramework
```

Comment:
Does this interface need the `ComImport` attribute to make sure it's embedded in projects that use the package, so the consuming project doesn't need to ship `NuGet.VisualStudio.dll`?